dask/dataframe/core.py

```diff
@@ -3232,6 +3233,17 @@ def __iter__(self):
             for row in s:
                 yield row
 
+    @_deprecated(
+        message=(
+            "Using the ``in`` operator to test for membership in Series is "
+            "deprecated. To test for membership in the index use "
+            "``(s.index == key).any()``. Similarly to test for membership in "
+            "the values use ``(s == key).any()``"
+        )
+    )
+    def __contains__(self, key):
+        return (self == key).any().compute()
```

Comment:
This matches the old behavior (aka does not match pandas).

Reply:
Is there a reason not to just have this return `(s.index == key).any()`? I.e. don't be eager but match pandas upon computing

Reply:
yeah I tried that one :( see https://github.com/dask/dask/pull/7914#discussion_r674079084